bin/adhoc-scripts/createPileupObjects.py

```diff
@@ -72,11 +73,13 @@ def getPUSchema(pileupName, pileupDocs, logger):
     :return: a dictionary with the pileup configuration
     """
     for idx, doc in enumerate(pileupDocs):
-        if pileupName == doc["pileupName"]:
+        pName = doc.get('customName', doc['pileupName'])
```

Comment:
I don't think this is needed, as the idea is to aggregate different documents using the same pileup name. I didn't check the whole code though, so I might be missing something.

Reply:
ok, revert it back.